release-material/maven-mobile/io/textile/mobile/VERSION/mobile-VERSION.pom

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.textile</groupId>
+  <artifactId>mobile</artifactId>
+  <version><VERSION></version>
+  <packaging>aar</packaging>
+  <name>mobile</name>
+  <description>Generated core API for Textile</description>
+  <url>https://github.com/textileio/go-textile</url>
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+  <developers>
+    <developer>
+      <id>textile</id>
+      <name>Textile</name>
+      <email>contact@textile.io</email>
+    </developer>
+  </developers>
+  <scm>
+    <connection>https://github.com/textileio/go-textile.git</connection>
+    <developerConnection>https://github.com/textileio/go-textile.git</developerConnection>
+    <url>https://github.com/textileio/go-textile</url>
+  </scm>
+</project>
```

Comment:
`<VERSION>` needs to be updated in there and in the path to here. Then this whole directory tree will be zipped up and uploaded to Bintray. Will post a link to their API docs.